workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/transform.py

```diff
@@ -17,6 +17,8 @@
 
 from .constants import HORIZONTAL_DIMS, DiagArg
 
+xr.set_options(keep_attrs=True)
```

Comment:
I get worried about this kind of global state/configuration making things magical. Can you scope this setting to the specific computing functions `time_mean`, `rms`, `bias`, etc?

Reply:
Done.